internal/pkg/agent/application/upgrade/rollback_test.go

```diff
@@ -328,6 +329,40 @@ func TestRollback(t *testing.T) {
 	}
 }
 
+func TestMakeBaseWatchCmd(t *testing.T) {
+	agentExecutable := "elastic-agent"
+	t.Run("no_rollback_window", func(t *testing.T) {
+		cmd := makeBaseWatchCmd(agentExecutable, 0)
+
+		// Expected command:
+		// elastic-agent watch --path.config /some/path --path.home /some/path
+		require.Len(t, cmd.Args, 6)
+		require.Equal(t, agentExecutable, cmd.Args[0])
+		require.Equal(t, "watch", cmd.Args[1])
+		require.Equal(t, "--path.config", cmd.Args[2])
+		require.NotEmpty(t, cmd.Args[3])
+		require.Equal(t, "--path.home", cmd.Args[4])
+		require.NotEmpty(t, cmd.Args[5])
```

Comment:
Nit: wouldn't this check be easier with a simple assert.Equal on the `cmd.Args` slice (same for the next test as well) ?
```suggestion
		expectedArgs := []string{agentExecutable, "watch", "--path.config", paths.Config(), "--path.home", paths.Home()}
		assert.Equal(t, expectedArgs, cmd.Args)
```

Reply:
Simplified in 50c2675057.